hasura.planx.uk/migrations/1715689232003_create view submission services log /up.sql

```diff
@@ -0,0 +1,45 @@
+create or replace view public.submission_services_log as 
+with payments as (
+    select 
+      session_id,
+      payment_id::text as event_id,
+      'Pay' as event_type,
+      initcap(status) as status,
+      '{}'::jsonb as response,
+      created_at
+    from payment_status
+    where status != 'created'
+      and created_at >= '2024-01-01'
```

Comment:
Is there any reason to query & display submissions prior to January 2024? This filter gives us a bit more consistency in event formatting & we could always look up submissions outside this timeframe on request.

Reply:
Totally on board with keeping this filter in place - makes a whole lot of sense.